src/PrestaShopBundle/Twig/TranslationsExtension.php

```diff
@@ -38,6 +38,15 @@ class TranslationsExtension extends \Twig_Extension
      */
     public $logger;
 
+    private $container;
```

Comment:
@mickaelandrieu It can not be as it would introduce a circular dependency exception (in the context of a Twig extension).